### PR TITLE
142 Add multi-gpu evaluation to spleen bundle

### LINKED ISSUE #142: Add multi-gpu evaluation config for spleen bundle
**Is your feature request related to a problem? Please describe.**
Currently, we only have multi-gpu config for the training, would be necessary to also add multi-gpu evaluation and multi-gpu inference. Raised by @holgerroth .

### PR DESCRIPTION
**Pull request by Nic-Ma**
Signed-off-by: Nic Ma <nma@nvidia.com>

Fixes #142 .

### Description
This PR added the multi-gpu evaluation config in the spleen bundle.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        ]
+    },
+    "validate#sampler": {
+        "_target_": "DistributedSampler",
+        "dataset": "@validate#dataset",
+        "even_divisible": false,
+        "shuffle": false
+    },
+    "validate#dataloader#sampler": "@validate#sampler",
+    "validate#handlers#1#_disabled_": "$True if dist.get_rank() > 0 else False",
```
Comment (wyli) [inline models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json]:
```suggestion
    "validate#handlers#1#_disabled_": "$dist.get_rank() > 0",
```

**Review by wyli — APPROVED** [review]

File: models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json (hunk 1 above)
Comment (Nic-Ma) [reply models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json]: Updated in the latest commit.

Thanks.

**Pull request merged**